pkg/controller/perconaservermongodb/statefulset.go

```diff
@@ -26,11 +26,14 @@ func (r *ReconcilePerconaServerMongoDB) reconcileStatefulSet(ctx context.Context
 	}
 
 	switch ls[naming.LabelKubernetesComponent] {
-	case "arbiter":
+	case naming.ComponentArbiter:
 		pdbspec = rs.Arbiter.PodDisruptionBudget
-	case "nonVoting":
+	case naming.ComponentNonVoting:
 		pdbspec = rs.NonVoting.PodDisruptionBudget
 		volumeSpec = rs.NonVoting.VolumeSpec
+	case naming.ComponentHidden:
```

Comment:
Given that we added a new case of replsets, I think it would be nice to have a new test case in our `statefulset_test.go` file, on the `TestReconcileStatefulSet` set. We have already covered these:

- mongod
- arbiter
- non-voting

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1925/commits/c5bbf87cd610a2ff050d2cec651aab599015ddad